cruz_solo_assessment/db/migrate/20150527134253_create_people.rb

```diff
@@ -0,0 +1,23 @@
+class CreatePeople < ActiveRecord::Migration
+  def change
+    create_table :people do |t|
+      t.text :name
+      t.text :title
+      t.text :quote
+      t.text :image
+      t.int :age
+
+      t.timestamps null: false
+    end
+  end
+end
+Daenerys Targaryen = Person.create(name: 'Daenerys Targaryen', title: 'Queen of Meereen and Mother of Dragons', quote: 'I will take what is mine with blood and fire.', image: 'http://i.imgur.com/DjHJq4E.jpg', age: 18
```

Comment:
I recommitted without it after I realized.

On Wed, May 27, 2015 at 2:54 PM, Max Blaushild <notifications@github.com>
wrote:

> In cruz_solo_assessment/db/migrate/20150527134253_create_people.rb
> <https://github.com/ga-wdi-boston/wdi_5_rails_crud_solo_assessment/pull/10#discussion_r31167748>
> :
>
> > @@ -0,0 +1,23 @@
> > +class CreatePeople < ActiveRecord::Migration
> > +  def change
> > +    create_table :people do |t|
> > +      t.text :name
> > +      t.text :title
> > +      t.text :quote
> > +      t.text :image
> > +      t.int :age
> > +
> > +      t.timestamps null: false
> > +    end
> > +  end
> > +end
> > +Daenerys Targaryen = Person.create(name: 'Daenerys Targaryen', title: 'Queen of Meereen and Mother of Dragons', quote: 'I will take what is mine with blood and fire.', image: 'http://i.imgur.com/DjHJq4E.jpg', age: 18
>
> Get this ish out of your migration file. Put it in either your scripts or
> your seed file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/wdi_5_rails_crud_solo_assessment/pull/10/files#r31167748>
> .
>